terraso_backend/apps/core/gis/parsers.py

```diff
@@ -0,0 +1,47 @@
+import json
+import os
+import uuid
+import zipfile
+
+import geopandas as gpd
+
+from apps.core.gis.utils import DEFAULT_CRS
+
+
+def isShapefile(file):
```

Comment:
suggestion: rename this function so the intent is more clear, i.e. you are only checking the extension, not the contents. perhaps hasShapefileExtension or isPossibleShapefile.